en-US/focus-ios.xliff

```diff
@@ -396,6 +419,11 @@
         <target>Open in %@</target>
         <note>Title for action sheet item to open the current page in another application. Placeholder is the name of the application to open the current page.</note>
       </trans-unit>
+      <trans-unit id="contextMenu.clipboardLink">
+        <source>Link you copied:</source>
+        <target>Link you copied:</target>
```

Comment:
Do you have a screenshot or UX mockup of how this string will be used? Is it really a context menu? How do you plan to insert the link?

If it's only one line, there should be a placeholder %@ for the link.